app/helpers/stages_helper.rb

```diff
@@ -26,4 +26,9 @@ def stage_lock_icon(stage)
       end
     content_tag :span, text.html_safe, class: "label label-warning", title: stage.lock.summary
   end
+
+  def stage_template_icon(stage)
+    return unless stage.is_template
+    image_tag "template-icon.png", size: "18x18"
```

Comment:
could add a title for the curious hoverers `title: "Template stage, this will be used when copying to to new deploy groups"`